website/docs/docs/introduction.md

```diff
@@ -3,188 +3,65 @@ title: "What is dbt?"
 id: "introduction"
 ---
 
-dbt enables analytics engineers to transform data in their warehouses by simply writing select statements. dbt handles turning these select statements into <Term id="table">tables</Term> and <Term id="view">views</Term>.
+## About dbt
 
-dbt does the `T` in <Term id="elt" /> (Extract, Load, Transform) processes – it doesn’t extract or load data, but it’s extremely good at transforming data that’s already loaded into your warehouse.
+<Snippet src="what-is-dbt-intro" />
 
-The role of dbt within a modern data stack is discussed in more detail [here](https://blog.fishtownanalytics.com/what-exactly-is-dbt-47ba57309068).
+dbt compiles and runs your analytics code against your data platform, enabling you and your team to collaborate on a single source of truth for metrics, insights, and business definitions. This single source of truth, combined with the ability to define tests for your data, reduces errors when logic changes, and alerts you when issues arise.
 
-dbt also enables analysts to work more like software engineers, in line with the dbt [Viewpoint](viewpoint).
+Read more about why we want to enable analysts to work more like software engineers,in [The dbt Viewpoint](/community/resources/viewpoint).
 
-## How do I use dbt?
+## dbt optimizes your workflow
 
-### dbt Projects
-A dbt project is a directory of `.sql` and .`yml` files. The directory must contain at a minimum:
-* Models: A model is a single `.sql` file. Each model contains a single `select` statement that either transforms raw data into a dataset that is ready for analytics, or, more often, is an intermediate step in such a transformation.
-* A project file: a `dbt_project.yml` file which configures and defines your dbt project.
+- Avoid writing boilerplate <Term id="dml" /> and <Term id="ddl" /> by managing transactions, dropping tables, and managing schema changes. Write business logic with just a SQL `select` statement that returns the dataset you need, and dbt takes care of <Term id="materialization" />.
```

Comment:
Would this be an instance where we need to mention Python or is the point being made not related?

Reply:
I think that would be awesome! Can you add something @matthewshaver ?